### ISSUE TITLE
Fails to launch on Mac 

### ISSUE THREAD

**Issue by DKroot** [jmeter-gradle-plugin/jmeter-gradle-plugin#32]
I am able to run `jmRun` on Windows (in Parallels), but exactly the same project fails to run on Mac with zero sampler results and no error message. Also, when I try to run jmGui on Mac, it fails with the following in the logs:

```
2015/10/27 12:18:01 INFO  - jmeter.util.JMeterUtils: Setting Locale to en_US 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: Copyright (c) 1998-2015 The Apache Software Foundation 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: Version 2.13 r1665067 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: java.version=1.8.0_20 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: java.vm.name=Java HotSpot(TM) 64-Bit Server VM 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: os.name=Mac OS X 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: os.arch=x86_64 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: os.version=10.9.5 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: file.encoding=UTF-8 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: Default Locale=English (United States) 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: JMeter  Locale=English (United States) 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: JMeterHome=/Users/DK/Projects/iRePORT/build/jmeter 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: user.dir  =/Users/DK/Projects/iRePORT/build/jmeter 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: PWD       =/Users/DK/Projects/iRePORT/build/jmeter 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: IP: 192.168.86.176 Name: Kiev.local FullName: 192.168.86.176 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: search_paths=/Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh-core/2.0b4/495e25a99e29970ffe8ba0b1d551e1d1a9991fc1/bsh-core-2.0b4.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_http/2.13/3c06ecdb5a4ac30449acea6487949a8c125fd547/ApacheJMeter_http-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit/2.13/98cebf69e1713e39d863cb77d614248db0a64a8c/ApacheJMeter_junit-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_java/2.13/aa892d72373c520fd589543a820de52b1fba775c/ApacheJMeter_java-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jdbc/2.13/65561a64e4afbac2a4dd63017169983c060121da/ApacheJMeter_jdbc-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_tcp/2.13/7f34411923b941ce7d8d7e9c700a7f7b73726989/ApacheJMeter_tcp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mail/2.13/b8ff5bc84886c02d03205262ddf90e7adf01e333/ApacheJMeter_mail-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ldap/2.13/31431bb6c19fab5342f216329a9c0bd37375fc50/ApacheJMeter_ldap-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ftp/2.13/232b83021cee7a3042ee4c62c503e8b926208cb4/ApacheJMeter_ftp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_functions/2.13/47c860e7e781ab8f327149471b425f4d33b20845/ApacheJMeter_functions-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_monitors/2.13/f10dd4066d5ddf91cf883bd71e8e409565a7afb5/ApacheJMeter_monitors-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jms/2.13/9d834b1884ab1f14605996ae9efd30bcf751a38e/ApacheJMeter_jms-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter/2.13/8b9e741eff0a94a5c5ee385ad2b1d9ab659c5bf9/ApacheJMeter-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_components/2.13/96e7bedc1e238e02b39063845171bca14525ff01/ApacheJMeter_components-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_config/2.13/5bd3bc1935fa85cf77d175afe9b8e63d4a2a4cff/ApacheJMeter_config-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_core/2.13/ac0fdf85299d4f61960cea49d7e8b3f9abe90fcd/ApacheJMeter_core-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit-test/2.13/32ac846495dd1aac8722ac45b16781ee37cbd823/ApacheJMeter_junit-test-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mongodb/2.13/5aa7d92c13f805ab74f5f9321551e3eabd716388/ApacheJMeter_mongodb-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_native/2.13/5d979b0348d2fa02deff9a10b9c735f8f3bca1b9/ApacheJMeter_native-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh/2.0b5/fdc2ab6ae8b53e0d4761b296c116df747cd85199/bsh-2.0b5.jar:/Users/DK/Projects/iRePORT/build/jmeter/lib/ext 
2015/10/27 12:18:01 WARN  - jmeter.JMeter: Can't read /Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh-core/2.0b4/495e25a99e29970ffe8ba0b1d551e1d1a9991fc1/bsh-core-2.0b4.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_http/2.13/3c06ecdb5a4ac30449acea6487949a8c125fd547/ApacheJMeter_http-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit/2.13/98cebf69e1713e39d863cb77d614248db0a64a8c/ApacheJMeter_junit-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_java/2.13/aa892d72373c520fd589543a820de52b1fba775c/ApacheJMeter_java-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jdbc/2.13/65561a64e4afbac2a4dd63017169983c060121da/ApacheJMeter_jdbc-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_tcp/2.13/7f34411923b941ce7d8d7e9c700a7f7b73726989/ApacheJMeter_tcp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mail/2.13/b8ff5bc84886c02d03205262ddf90e7adf01e333/ApacheJMeter_mail-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ldap/2.13/31431bb6c19fab5342f216329a9c0bd37375fc50/ApacheJMeter_ldap-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ftp/2.13/232b83021cee7a3042ee4c62c503e8b926208cb4/ApacheJMeter_ftp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_functions/2.13/47c860e7e781ab8f327149471b425f4d33b20845/ApacheJMeter_functions-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_monitors/2.13/f10dd4066d5ddf91cf883bd71e8e409565a7afb5/ApacheJMeter_monitors-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jms/2.13/9d834b1884ab1f14605996ae9efd30bcf751a38e/ApacheJMeter_jms-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter/2.13/8b9e741eff0a94a5c5ee385ad2b1d9ab659c5bf9/ApacheJMeter-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_components/2.13/96e7bedc1e238e02b39063845171bca14525ff01/ApacheJMeter_components-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_config/2.13/5bd3bc1935fa85cf77d175afe9b8e63d4a2a4cff/ApacheJMeter_config-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_core/2.13/ac0fdf85299d4f61960cea49d7e8b3f9abe90fcd/ApacheJMeter_core-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit-test/2.13/32ac846495dd1aac8722ac45b16781ee37cbd823/ApacheJMeter_junit-test-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mongodb/2.13/5aa7d92c13f805ab74f5f9321551e3eabd716388/ApacheJMeter_mongodb-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_native/2.13/5d979b0348d2fa02deff9a10b9c735f8f3bca1b9/ApacheJMeter_native-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh/2.0b5/fdc2ab6ae8b53e0d4761b296c116df747cd85199/bsh-2.0b5.jar:/Users/DK/Projects/iRePORT/build/jmeter/lib/ext 
2015/10/27 12:18:01 INFO  - jmeter.gui.action.LookAndFeelCommand: Using look and feel: com.apple.laf.AquaLookAndFeel [Mac OS X, System] 
2015/10/27 12:18:01 INFO  - jmeter.JMeter: Loaded icon properties from org/apache/jmeter/images/icon.properties 
2015/10/27 12:18:02 FATAL - jmeter.gui.action.ActionRouter: !!!!!Uh-oh, didn't find any action handlers!!!!! 
2015/10/27 12:18:02 FATAL - jmeter.JMeter: An error occurred:  org.apache.jorphan.util.JMeterError: No action handlers found - check JMeterHome and libraries
    at org.apache.jmeter.gui.action.ActionRouter.populateCommandMap(ActionRouter.java:301)
    at org.apache.jmeter.gui.action.ActionRouter.getInstance(ActionRouter.java:332)
    at org.apache.jmeter.JMeter.startGui(JMeter.java:239)
    at org.apache.jmeter.JMeter.start(JMeter.java:385)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at org.apache.jmeter.NewDriver.main(NewDriver.java:264)
```

**Comment by foragerr** [comment]
Interesting, works on windows but not on Mac? I imagine I messed up a path separator somewhere. I'll take a look at it.

**Comment by foragerr** [comment]
So the problem could be this:

```
2015/10/27 12:18:01 INFO  - jmeter.JMeter: search_paths=/Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh-core/2.0b4/495e25a99e29970ffe8ba0b1d551e1d1a9991fc1/bsh-core-2.0b4.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_http/2.13/3c06ecdb5a4ac30449acea6487949a8c125fd547/ApacheJMeter_http-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit/2.13/98cebf69e1713e39d863cb77d614248db0a64a8c/ApacheJMeter_junit-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_java/2.13/aa892d72373c520fd589543a820de52b1fba775c/ApacheJMeter_java-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jdbc/2.13/65561a64e4afbac2a4dd63017169983c060121da/ApacheJMeter_jdbc-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_tcp/2.13/7f34411923b941ce7d8d7e9c700a7f7b73726989/ApacheJMeter_tcp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mail/2.13/b8ff5bc84886c02d03205262ddf90e7adf01e333/ApacheJMeter_mail-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ldap/2.13/31431bb6c19fab5342f216329a9c0bd37375fc50/ApacheJMeter_ldap-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ftp/2.13/232b83021cee7a3042ee4c62c503e8b926208cb4/ApacheJMeter_ftp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_functions/2.13/47c860e7e781ab8f327149471b425f4d33b20845/ApacheJMeter_functions-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_monitors/2.13/f10dd4066d5ddf91cf883bd71e8e409565a7afb5/ApacheJMeter_monitors-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jms/2.13/9d834b1884ab1f14605996ae9efd30bcf751a38e/ApacheJMeter_jms-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter/2.13/8b9e741eff0a94a5c5ee385ad2b1d9ab659c5bf9/ApacheJMeter-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_components/2.13/96e7bedc1e238e02b39063845171bca14525ff01/ApacheJMeter_components-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_config/2.13/5bd3bc1935fa85cf77d175afe9b8e63d4a2a4cff/ApacheJMeter_config-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_core/2.13/ac0fdf85299d4f61960cea49d7e8b3f9abe90fcd/ApacheJMeter_core-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit-test/2.13/32ac846495dd1aac8722ac45b16781ee37cbd823/ApacheJMeter_junit-test-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mongodb/2.13/5aa7d92c13f805ab74f5f9321551e3eabd716388/ApacheJMeter_mongodb-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_native/2.13/5d979b0348d2fa02deff9a10b9c735f8f3bca1b9/ApacheJMeter_native-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh/2.0b5/fdc2ab6ae8b53e0d4761b296c116df747cd85199/bsh-2.0b5.jar:/Users/DK/Projects/iRePORT/build/jmeter/lib/ext 
2015/10/27 12:18:01 WARN  - jmeter.JMeter: Can't read /Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh-core/2.0b4/495e25a99e29970ffe8ba0b1d551e1d1a9991fc1/bsh-core-2.0b4.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_http/2.13/3c06ecdb5a4ac30449acea6487949a8c125fd547/ApacheJMeter_http-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit/2.13/98cebf69e1713e39d863cb77d614248db0a64a8c/ApacheJMeter_junit-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_java/2.13/aa892d72373c520fd589543a820de52b1fba775c/ApacheJMeter_java-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jdbc/2.13/65561a64e4afbac2a4dd63017169983c060121da/ApacheJMeter_jdbc-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_tcp/2.13/7f34411923b941ce7d8d7e9c700a7f7b73726989/ApacheJMeter_tcp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mail/2.13/b8ff5bc84886c02d03205262ddf90e7adf01e333/ApacheJMeter_mail-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ldap/2.13/31431bb6c19fab5342f216329a9c0bd37375fc50/ApacheJMeter_ldap-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_ftp/2.13/232b83021cee7a3042ee4c62c503e8b926208cb4/ApacheJMeter_ftp-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_functions/2.13/47c860e7e781ab8f327149471b425f4d33b20845/ApacheJMeter_functions-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_monitors/2.13/f10dd4066d5ddf91cf883bd71e8e409565a7afb5/ApacheJMeter_monitors-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_jms/2.13/9d834b1884ab1f14605996ae9efd30bcf751a38e/ApacheJMeter_jms-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter/2.13/8b9e741eff0a94a5c5ee385ad2b1d9ab659c5bf9/ApacheJMeter-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_components/2.13/96e7bedc1e238e02b39063845171bca14525ff01/ApacheJMeter_components-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_config/2.13/5bd3bc1935fa85cf77d175afe9b8e63d4a2a4cff/ApacheJMeter_config-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_core/2.13/ac0fdf85299d4f61960cea49d7e8b3f9abe90fcd/ApacheJMeter_core-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_junit-test/2.13/32ac846495dd1aac8722ac45b16781ee37cbd823/ApacheJMeter_junit-test-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_mongodb/2.13/5aa7d92c13f805ab74f5f9321551e3eabd716388/ApacheJMeter_mongodb-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.apache.jmeter/ApacheJMeter_native/2.13/5d979b0348d2fa02deff9a10b9c735f8f3bca1b9/ApacheJMeter_native-2.13.jar:/Users/DK/.gradle/caches/modules-2/files-2.1/org.beanshell/bsh/2.0b5/fdc2ab6ae8b53e0d4761b296c116df747cd85199/bsh-2.0b5.jar:/Users/DK/Projects/iRePORT/build/jmeter/lib/ext 
```

jmeter apparently cant read the `search_paths` supplied. The path separator on windows is a `;` and `:` on linux. I would imagine a Mac uses the same path separator as linux? would you know if this is the case?

edit: Google searches indicate macs do use `:` as a path separator.

**Comment by foragerr** [comment]
Gah, forgot that [jmeter uses `;` as a path separator for `search_paths` irrespective of platform](http://jmeter.apache.org/usermanual/get-started.html#configuring_jmeter). The change to fix this [is already pulled into source](https://github.com/jmeter-gradle-plugin/jmeter-gradle-plugin/commit/9a34ef9c2b9a8079e1a2d33954b1d9315ac8aabe). Will do a new release soon.

**Comment by DKroot** [comment]
Thank you!

**Comment by foragerr** [comment]
Released v0.6.3 to bintray.
